projects/vdk-plugins/vdk-meta-jobs/README.md

```diff
@@ -0,0 +1,146 @@
+# Meta Jobs
+
+Express dependecies between data jobs.
+
+A plugin for Versatile Data Kit which extends its Job API with an additional feature which allows users to trigger so called Meta Jobs.
+Meta Jobs are data jobs which trigger one or more other jobs, wait for their completion, and then trigger another set of jobs until either the entire job pipeline has succeeded.
+
+
+## Usage
+
+```
+pip install vdk-meta-jobs
+```
+
+Then one would create a single step and define the jobs we want to orchestrate
+
+```python
+def run(job_input):
+    jobs = [
+        {
+        "job_name": "name-of-job",
+        "team_name": "team-of-job",
+        "fail_meta_job_on_error": True or False,
+        "depends_on": [name-of-job1, name-of-job2]
+        },
+        ...
+    ]
+    MetaJobInput().run_meta_job(jobs)
+```
+
+When defining a job to be run following attributes are supported:
+* **job_name**: required, the name of the data job
+* **team_name:**: optional, the team of the data job. If omitted , it will use the meta job's team
+* **fail_meta_job_on_error**: optional, default is true. if true, the meta job will abort and fail if the orchestrated job fails, if false, meta job won't fail and continue.
+* **depends_on**: required (can be empty), list of other jobs that the orchestrated job depends on. The job will not be started until depends_on job have finished.
+
+
+### Example
+
+The following example dependency graph can be implemented with below code.
```

Comment:
I believe a clarification about concurrency should be added. My assumption for this specific example is the following:
1. Job 1 will execute
2. After Job 1 is completed, jobs 2,3,4 will start executing in parallel
3. Jobs 5 and 6 will start executing after job 3 completes, but will not wait for the completion of jobs 2 and 4.

Is this assumption correct?